init.rb

```diff
@@ -1,6 +1,8 @@
 require 'redmine_s3'
 
-Redmine::Plugin.register :redmine_s3_attachments do
+require_dependency 'redmine_s3_hooks'
+
+Redmine::Plugin.register :redmine_s3 do
```

Comment:
Yes, there is. I added a script file and it was not copied to public/plugin_assets because plugin name was not the same as its folder. If it is named 'redmine_s3' I don't see why should it be registered under different name.